src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../Movie/Movie';
+
+export const MovieCard: React.FC<Movie> = ({ ...movie }) => (
```

Comment:
Don't use rest operator to collect object from props.
And create type `Props` here.